nupic/research/frameworks/pytorch/cifar_experiment.py

```diff
@@ -20,54 +20,70 @@
 # Original Code here:
 # https://github.com/pytorch/examples/blob/master/mnist/main.py
 
-import argparse
-import configparser
 import time
 
-import ray
+import torch
 import torch.nn as nn
-import torch.optim as optim
 from nupic.torch.modules import (KWinners, SparseWeights, Flatten, KWinners2d, rezeroWeights, updateBoostStrength)
-from ray import tune
 from torchvision import datasets, transforms
 from torch.utils.data import DataLoader
 
 from nupic.research.frameworks.pytorch.model_utils import *
 from nupic.research.frameworks.pytorch.image_transforms import *
 
+
 def CNNSize(width, kernel_size, padding=1, stride=1):
   return (width - kernel_size + 2 * padding) / stride + 1
 
 
-class TinyCIFAR(tune.Trainable):
+class TinyCIFAR(object):
   """
-  ray.tune trainable class:
-  - Override _setup to reset the experiment for each trial.
-  - Override _train to train and evaluate each epoch
-  - Override _save and _restore to serialize the model
+  Generic class for creating tiny CIFAR models. This can be used with Ray tune
```

Comment:
Nice! Maybe later we could refactor into a generic interface for any tunable experiment, not just CIFAR

Reply:
Good idea. We should do that (in a separate PR).